fleet_packages.json

```diff
@@ -0,0 +1,41 @@
+/*
+
+  Packages in this file are considered "bundled" and are installed as part of Fleet's setup process. Each entry points to a valid version name
+  avaiable in the Elastic Package Registry service, and must include a sha-512 checksum of the `.zip` archive for the given package.
+
+  When adding or updating a package to this file, download the appropriate `.zip` archive from the EPR service,
+  e.g. "https://epr-snapshot.elastic.co/epr/apm/apm-8.1.0-dev2.zip` and generate a checksum from that `.zip`, e.g:
+
+  ```
+  $ shasum -a 512 ./synthetics-0.8.0.zip
```

Comment:
Just a heads-up, with Package Storge v2, the package registry will serve package signatures (signed with Elastic private key). You may want to leave a space for that.

Reply:
Do you think this could replace the initial checksum process here, or would this be an additional field?

Reply:
It's [already in the Package Registry](https://github.com/elastic/package-registry/blob/8c5627a065fde31ff819026aaca781514cfa0827/testdata/generated/package-zip.json#L22), its logic is ready for it. We are just not populating it yet.